app/components/Channels/ChannelNodeSearch.js

```diff
@@ -61,7 +62,8 @@ class ChannelNodeSearch extends React.PureComponent {
    * Handle connect to node click.
    */
   handleClickConnect = node => {
-    this.formApi.setValue('search', `${node.pub_key}@${node.addresses[0].addr}`)
+    const { updateContactFormSearchQuery } = this.props
+    updateContactFormSearchQuery(`${node.pub_key}@${node.addresses[0].addr}`)
```

Comment:
Could you please add a debounce out of the box?

Reply:
I deliberately didn't add a debounce here, as it just makes the UI unnecessarily unresponsive. This is the `Connect` button. When the user clicks on the button the app should respond instantly, not in 300 ms.

Reply:
Sorry, it's ok. Haven't seen a method name. Search is already debounced here